<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mako D1 - MidiControl</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openMidiPortButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="midiChannelSpinBox">
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>760</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>12</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>760</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="ProgramcomboBox">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>416</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Digital</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vintage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reverse</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="presetToggleButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>224</red>
          <green>27</green>
          <blue>36</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>224</red>
          <green>27</green>
          <blue>36</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>239</green>
          <blue>239</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>153</red>
          <green>193</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="presetDial">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>140</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="presetlcd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="timeDial">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="repeatsDial">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QDial" name="mixDial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QDial" name="ageDial">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QDial" name="toneDial">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QDial" name="modDial">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QDial" name="attackDial">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>64</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeats</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mix</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mod</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tone</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="DivsioncomboBox">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>446</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Quarter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eighth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dotted Eighth</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Division</string>
    </property>
   </widget>
   <widget class="QComboBox" name="midiDevicecomboBox">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>90</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="presetUpButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-up"/>
    </property>
   </widget>
   <widget class="QPushButton" name="presetDownButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-down"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
